docs/VnVPlan/VnVPlan.tex

```diff
@@ -216,57 +216,616 @@ \section{System Test Description}
+                        
+      Initial State: The game is in Engine Mode.
+                          
+      Input: The system shall use the chess engine to evaluate the position and calculate the best
+      engine moves.
+                          
+      Output: The system shall display the calculated engine moves on the web application.
+                          
+      Test Case Derivation: The chess engine should update the board configuration after receiving new information about the board state. 
+  
+      How test will be performed: The function that calculates the engine moves
+      will be run using the appropriate inputs. After it has ran we will check to see if the correct engine moves are played
+      by comparing the results to the identical engine on an online platform.
+  
+      \item{EA-6\\}
+  
+      Type: Functional, Dynamic, Manual
+                        
+      Initial State: The game is in Engine Mode.
+                          
+      Input: The game has been terminated through method of stalemate, checkmate, resignation, or draw.
+                          
+      Output: The web application will display a message with the method of game termination and the
+      system shall change to the Game Inactive State.
+                          
+      Test Case Derivation: The web application should display a message signifying user of the method of game termination
+      and the game should modify state if the game has ended.
+  
+      How test will be performed: The function that checks to see if any of the game termination methods are in effect
+      will be run using the appropriate inputs. After it has ran we will out the termination method of change the
+      game state to the Game Inactive State.
+  
+    \end{enumerate}
+     
+    \subsubsection{Beginner Mode}
+  
+    \begin{enumerate}
+  
+      \item{BB-1\\}
+  
+      Type: Functional, Dynamic, Manual
+                          
+      Initial State: The game is in Beginner Mode.
+                          
+      Input: A piece has been moved to a square.
+                          
+      Output: The system stores the position, colour, and type of piece in the micro-controller.
+                          
+      Test Case Derivation: The system should contain the position, colour, and type in the micro-controller
+      after a piece has been moved to a square.
+  
+      How test will be performed: The function that accesses the micro-controller will be run using the appropriate inputs.
+      After it has ran we will check to see if the position, colour, and type of piece are stored in the micro-controller.  
+  
+      \item{BB-2\\}
+  
+      Type: Functional, Dynamic, Manual
+                          
+      Initial State: The game is in Beginner Mode.
+                          
+      Input: The user picks up a piece.
+                          
+      Output: Allow user to view all legal moves with green LED
```

Comment:
Maybe word the output like "User views all legal ..." instead. Allowing seems like a weird output. 
Also maybe put "inputs and outputs at the end of the first sentence in this How test will be performed: section